src/kbmod/analysis/wcs_utils.py

```diff
@@ -0,0 +1,143 @@
+import astropy.wcs
+import astropy.units
+import astropy.coordinates
+import numpy
+
+# if True then use when image_fov is provided with no pixel_scale solve for a pixel_scale that results in the image_fov
+solve_for_image_fov = False
```

Comment:
I will change. I hope that we can replace this with an analytic solution since it seems like it should be something like
pixel_scale*pixels = 2*tan(image_fov/2) but that didn't work out for me. Nor did pixel_scale*pixels=image_fov (which is in the code).

Reply:
yes. done